rpcs3/Emu/Cell/Modules/libmixer.cpp

```diff
@@ -6,6 +6,13 @@
 #include "cellAudio.h"
 #include "libmixer.h"
 
+#if __GNUC__ >= 6
+#include <math.h>
+#define FABS(X) std::fabs(X)
```

Comment:
We already require c++14 support (at least partial) and add it to flags: https://github.com/RPCS3/rpcs3/blob/master/rpcs3/CMakeLists.txt#L42. We don't test/support ICC, and the example seems to compile on clang 3.5+ and gcc 4.9+ just fine.